artiq/gateware/drtio/wrpll/si549.py

```diff
@@ -296,8 +315,10 @@ def __init__(self, pads):
                 ts_scl.o.eq(self.gpio_out.storage[0]),
                 ts_scl.oe.eq(self.gpio_oe.storage[0])
             ).Else(
-                ts_scl.o.eq(programmer.scl),
-                ts_scl.oe.eq(1)
+                #ts_scl.o.eq(programmer.scl),
+                #ts_scl.oe.eq(1)
```

Comment:
Remove commented code.
Also as I pointed out earlier, this change is acceptable on its own and should go in a separate pull request.